### PR TITLE
Create a new project for the xgov interface

### PR DESCRIPTION
**Pull request by ricdikulous**

### CODE REVIEW

File: src/xgov-dapp/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script>
+      window.global = window
+    </script>
+    <link rel="icon" type="image/png" href="/images/algorand.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Algorand Voting Tool</title>
```
Comment (robdmoore) [inline src/xgov-dapp/index.html:11]: Minor: This should probably reference xGov

File: src/xgov-dapp/src/features/vote-creation/RoundInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                  toISO: (date) => dayjs(date).toISOString(),
+                  fromISO: (date) => dayjs(date) as unknown as Date,
+                })}
+              </div>
+              {helper.textFileFormField({
+                label: 'Proposals',
+                field: 'proposalFile',
+                hint: 'Upload a .csv file',
+                longHint:
+                  'Upload a CSV file with 5 columns containing the snapshots. The headers should be "title", "description", "link", "category", "threshold"',
```
Comment (robdmoore) [inline src/xgov-dapp/src/features/vote-creation/RoundInfo.tsx:161]:
```suggestion
                  'Upload a CSV file with 5 columns containing the proposals. The headers should be "title", "description", "link", "category", "threshold"',
```

Comment (robdmoore) [inline src/xgov-dapp/src/features/vote-creation/RoundInfo.tsx:161]:
```suggestion
                  'Upload a CSV file with 5 columns containing the proposals. The headers should be "title", "description", "link", "category", "threshold"',
```

File: src/xgov-dapp/src/features/vote-creation/review/index.tsx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+import { ProposalCard } from '../../../shared/ProposalCard'
+import { VotingRoundGlobalState } from '../../../shared/VotingRoundContract'
+import { Proposal } from '../../../shared/types'
+import { VotingTime } from '../../vote/VotingTime'
+
+export default function Review() {
+  const [confirmationDialogOpen, setConfirmationDialogOpen] = useState(false)
+  const { activeAddress, signer: transactionSigner } = useWallet()
+  const roundInfo = useRoundInfo()
+  const proposalsParsed = Papa.parse<Proposal>(roundInfo.proposalFile, { header: true }).data
```
Comment (robdmoore) [inline src/xgov-dapp/src/features/vote-creation/review/index.tsx:32]: What happens if the csv is corrupt, do we handle it nicely?

**Review by robdmoore — APPROVED** [review]

File: src/xgov-dapp/src/features/vote-creation/review/index.tsx (hunk 3 above)
Comment (ricdikulous) [reply src/xgov-dapp/src/features/vote-creation/review/index.tsx:32]: There's pretty full on validation on the previous screen but yeah I can make it more defensive here too. Will address in a separate PR though

**Pull request merged**